packages/forms/src/elements/currency/currency.tsx

```diff
@@ -215,6 +224,7 @@ const InputCurrency: React.FC<InputCurrencyProps> = React.memo(
 					width={width}
 					testId={testId}
 					label={label}
+					ariaLabelSuffix={i18n.ariaLabelExtension}
```

Comment:
It would be good to have consistent naming here.

Reply:
Agree, and I've homogenised the names to 'ariaLabelExtension'